test/dotnet.Tests/CommandTests/Tool/Install/ToolInstallLocalCommandTests.cs

```diff
@@ -128,22 +128,41 @@ public void WhenRunWithPackageIdItShouldSaveToCacheAndAddToManifestFile()
         }
 
         [Fact]
-        public void GivenNoManifestFileItShouldThrow()
+        public void GivenCreateManifestIfNeededWithoutArgumentTheDefaultIsTrueForLegacyBehavior()
         {
             _fileSystem.File.Delete(_manifestFilePath);
-            var toolInstallLocalCommand = GetDefaultTestToolInstallLocalCommand();
+            ParseResult parseResult =
+            Parser.Instance.Parse(
+               $"dotnet tool install {_packageIdA.ToString()} --create-manifest-if-needed");
 
-            Action a = () => toolInstallLocalCommand.Execute();
-            a.Should().Throw<GracefulException>()
-                .And.Message.Should()
-                .Contain(CliStrings.CannotFindAManifestFile);
+            var toolInstallLocalCommand = new ToolInstallLocalCommand(
+                parseResult,
+                _packageIdA,
+                _toolPackageDownloaderMock,
+                _toolManifestFinder,
+                _toolManifestEditor,
+                _localToolsResolverCache,
+                _reporter);
+
+            toolInstallLocalCommand.Execute().Should().Be(0);
         }
 
         [Fact]
         public void GivenNoManifestFileItShouldThrowAndContainNoManifestGuide()
         {
             _fileSystem.File.Delete(_manifestFilePath);
-            var toolInstallLocalCommand = GetDefaultTestToolInstallLocalCommand();
+            ParseResult parseResult =
+            Parser.Instance.Parse(
+               $"dotnet tool install {_packageIdA.ToString()} --create-manifest-if-needed false");
```

Comment:
Should we update the text for ToolInstallNoManifestGuide?

> If you intended to install a global tool, add `--global` to the command.
> If you would like to create a manifest, use the `--create-manifest-if-needed` flag with the `dotnet tool install` command, or use `dotnet new tool-manifest`, usually in the repo root directory.

Reply:
That's a thoughtful question and I think worth considering. We could remove the --create-manifest-if-needed part, though I think that obfuscates the fact that there is a manifest as a concept when people hit this error. There may be a lot of users who aren't aware of don't care about the manifest, IMO, so maybe it's good to keep this message as is. It is not misleading/incorrect despite this. I'm also not sure when this will be shown now, besides when using false for create manifest if needed in the old scenarios.